NorthstarDedicatedTest/logging.cpp

```diff
@@ -10,10 +10,12 @@
 #include <Psapi.h>
 #include <minidumpapiset.h>
 #include "configurables.h"
+#include "logrotation.h"
 
 // This needs to be called after hooks are loaded so we can access the command line args
 void CreateLogFiles()
 {
+	logRotation(7);
```

Comment:
This should not happen if logs are disabled, and there should also be a separate option to disable log rotation (possibly passing it the number of days, with 0 being disabled).